jazzy/distribution.yaml

```diff
@@ -7637,6 +7637,33 @@ repositories:
       url: https://github.com/PickNikRobotics/rviz_visual_tools.git
       version: ros2
     status: maintained
+  scenario_execution:
+    doc:
+      type: git
+      url: https://github.com/IntelLabs/scenario_execution.git
+      version: jazzy
+    release:
+      packages:
+      - scenario_coverage
+      - scenario_execution
+      - scenario_execution_control
+      - scenario_execution_gazebo
+      - scenario_execution_interfaces
+      - scenario_execution_nav2
+      - scenario_execution_os
+      - scenario_execution_py_trees_ros
+      - scenario_execution_ros
+      - scenario_execution_rviz
+      - scenario_execution_x11
+      tags:
+        release: release/jazzy/{package}/{version}
+      url: https://github.com/ros2-gbp/scenario_execution-release.git
+      version: 1.2.0-1
+    source:
+      type: git
+      url: https://github.com/IntelLabs/scenario_execution.git
+      version: main
```

Comment:
I think the source version should refer to the `jazzy` branch instead of the main.